cns/types/infiniband/status.go

```diff
@@ -0,0 +1,11 @@
+package infiniband
+
+type Status int
+
+const (
+	ProgrammingPending  Status = 0
+	ProgrammingFailed   Status = 1
+	ProgrammingComplete Status = 2
+	ReleasePending      Status = 3
+	Available           Status = 4
```

Comment:
thinking if this should be 0? we should have a default state & once released we should let IB status reflect that initial state, since we aren't actually deleting an IB here just marking it available or programmed

Reply:
Done